### PR TITLE
Add batchnorm running calc weight to porting script

### PR DESCRIPTION
**Pull request by amyeroberts**
# What does this PR do?

Add two weight name mappings for PyTorch -> TensorFlow necessary for cross-loading weights for batchnorm layers which have been trained with `tracking_running_stats=True`.

This was necessary for cross-loading weights for the ResNet and RegNet ports. #17536 , #17554 

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?
NB. I couldn't find tests corresponding to the current weight loading logic.

## Who can review?

Anyone in the community is free to review the PR once the tests have passed.

### CODE REVIEW

**Review by sgugger — COMMENTED** [review]
Thanks a lot for diving into this. I would very very very much prefer if we could that rename in the functions that change the weights (and maybe reshape) as this renaming gamma/beta has been a huge pain in the butt for many contributors (and we actually wanted to get rid of it when we find a way to make it in a BC fashion).

**Comment by sgugger** [comment]
Actually, I misread the file where it's modified 😅 
It's fine for the conversion like this, it's the code in modeling_utils that does this I don't want (like [here](https://github.com/huggingface/transformers/blob/9fc34235fa3329c918d5ba67ce09a0cc8f399c59/src/transformers/modeling_utils.py#L432). Sorry I didn't pay close enough attention.
